tests/unit/src/Internal/Payment/State/InitialStateTest.php

```diff
@@ -90,41 +97,8 @@ protected function setUp(): void {
 			$this->mock_payment_request_service
 		);
 		$this->sut->set_context( $this->mock_context );
-	}
-
-	/**
-	 * Different `process` scenarios.
-	 *
-	 * There will be a single parameter, representing an exception, if any.
-	 *
-	 * @return array
-	 */
-	public function provider_process() {
-		return [
-			[ null ],
-			[ new Invalid_Request_Parameter_Exception( 'A parameter cannot be set.', 'invalid_parameter' ) ],
-			[ new Extend_Request_Exception( 'A parameter cannot be set.', 'cannot_extend' ) ],
-		];
-	}
 
-	/**
-	 * Ensures that the `process` method creates a new completed state.
-	 *
-	 * @param \Exception|null $exception Exception that would be thrown by intent creation.
-	 * @dataProvider provider_process
-	 */
-	public function test_process( Exception $exception = null ) {
-		$order_id     = 123;
-		$mock_request = $this->createMock( PaymentRequest::class );
-		$mock_intent  = $this->createMock( WC_Payments_API_Payment_Intention::class );
-
-		/**
-		 * This test works with the root `process` method, which calls a few
-		 * internal methods. We want to mock them for the purpose of this test.
-		 *
-		 * @var MockObject|InitialState
-		 */
-		$this->sut = $this->getMockBuilder( InitialState::class )
+		$this->mocked_sut = $this->getMockBuilder( InitialState::class )
```

Comment:
We still need the comment with the `@var` declaration here, at least for VS Code: 
<img width="779" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/5311119/1c443e84-9def-4bad-b8fc-98ddbb19004f">

Even though the private property is defined as `InitialState|MockObject`, the definition here makes the editor thing it's just a `MockObject`

You can simply restore the comment that was there:
```suggestion
		/**
		 * This test works with the root `process` method, which calls a few
		 * internal methods. We want to mock them for the purpose of this test.
		 *
		 * @var MockObject|InitialState
		 */
		$this->mocked_sut = $this->getMockBuilder( InitialState::class )
```